Network/src/Game/game_tick.c

```diff
@@ -0,0 +1,52 @@
+/*
+** EPITECH PROJECT, 2024
+** Zappy_Server
+** File description:
+** No file there , just an epitech header example .
+*/
+
+#include "Server/server.h"
+#include <sys/time.h>
+
+static uint64_t get_time(void)
+{
+    struct timeval tv;
+
+    gettimeofday(&tv, NULL);
+    return tv.tv_sec * (uint64_t)1000000 + tv.tv_usec;
+}
+
+static void spawn_resources(server_t *server)
+{
+    int x;
+    int y;
+
+    for (int k = 0; k < MAX_ITEMS; k++) {
+        for (int i = 0; i < server->game->picked_up_items[k] &&
+        server->game->picked_up_items[k] > 0; i++) {
```

Comment:
variable has to be defined at the top of the function

```suggestion
    int x;
    int y;
    int k;
    int i;

    for (k = 0; k < MAX_ITEMS; k++) {
        for (i = 0; i < server->game->picked_up_items[k] &&
        server->game->picked_up_items[k] > 0; i++) {
```

Reply:
not in instantiation of a for loop